### ISSUE TITLE
bug: Something is wrong with the video titles 

### ISSUE THREAD

**Issue by Znackt** [code100x/cms#1635]
Description:
There is a glitch that video titles aren't visible, I tried on different networks but I still have the same issue

Below is the video for this issue : 

https://github.com/user-attachments/assets/997cb0bb-c331-4285-8ae3-c774fb1d8457

** Browser specs **
 - Browser [ Microsoft Edge ]
 - Version [ 131.0.2903.86 ]

**Comment by pkmanas22** [comment]
Can i raise PR, i made changes, here is the screenshot
![image](https://github.com/user-attachments/assets/bd554ff4-bcfd-43d1-a1d2-ce577627dcf1)

**Comment by pkmanas22** [comment]
@devsargam sir, should I make PR, or wait for @Znackt 's reply

**Comment by IkramBagban** [comment]
> @devsargam sir, should I make PR, or wait for @Znackt 's reply

someone has already raised a pr for this

**Comment by pkmanas22** [comment]
> > @devsargam sir, should I make PR, or wait for @Znackt 's reply
> 
> someone has already raised a pr for this

Yeah PR is there, but I think that one is not pointing to any issue, he may directly raise PR, without any issue

**Comment by Znackt** [comment]
> > > @devsargam sir, should I make PR, or wait for @Znackt 's reply
> > 
> > 
> > someone has already raised a pr for this
> 
> Yeah PR is there, but I think that one is not pointing to any issue, he may directly raise PR, without any issue

If it's fixed , you may raise a PR

**Comment by pkmanas22** [comment]
Okay then
